esda/moran.py

```diff
@@ -889,6 +875,9 @@ class Moran_Local(object):
         value to use as a weight for the "fake" neighbor for every island. If numpy.nan,
         will propagate to the final local statistic depending on the `stat_func`. If 0, then
         the lag is always zero for islands.
+    alternative: string
```

Comment:
```suggestion
    alternative: {"two-tailed", "one-tailed"}
        (default="two-tailed")
```

This follows the pattern used above.